web/public/mgmt/config.yml

```
@@ -3120,8 +3120,8 @@ collections:
                   widget: text,
                 }
               - {
-                  label: Opportunities and Deadlines Snackbar Time Delay,
-                  name: opportunitiesAndDeadlinesTimeDelay,
+                  label: Time Delay Between Multiple Snackbars,
+                  name: timeDelayBetweenMultipleSnackbars,
```

[sand] Add a descriptor here to indicate what this value represents.

Might be worth adding a note as well. If we're surfacing this in the CMS then the content team should know what this is.
```suggestion
                  label: Time Delay Between Multiple Snackbars (time in milliseconds),
                  name: timeDelayBetweenMultipleSnackbars,
```

---

I'd check with them for specific messaging for the note.